src/main/scala/org/dberg/hubot/listeners/TestListener3.scala

```diff
@@ -0,0 +1,21 @@
+package org.dberg.hubot.listeners
+
+import org.dberg.hubot.Hubot
+import org.dberg.hubot.listeners.Listener.CallbackSuccess
+import org.dberg.hubot.models._
+
+class TestListener3(hubot: Hubot) extends Listener(hubot) {
+
+  val callback: Callback = {
```

Comment:
Thinking about this too.  with the partial Function return type we lose the CallbackResult type which allows us to properly test for failure conditions properly no ?  [Message,Unit] makes it harder in tests no ?